### ISSUE TITLE
[BUG] <title>

### ISSUE THREAD

**Issue by jolevesq** [Canadian-Geospatial-Platform/geoview#1391]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The popover are not WCAG, when we tab to button to open, focus is  not in the notification panel and there is no way to close it

### Expected Behavior

We should be able to toggle open close of our 2 popover notification and version

### Steps To Reproduce

1. On any pages, tab to the button
2. Press enter or space
3. Try to close or access the content

**Comment by jolevesq** [comment]
Duplicate of #1322
